test-app/tests/integration/rendering-test.ts

```diff
@@ -9,6 +9,12 @@ module('Integration | Rendering', function (hooks) {
   setupRenderingTest(hooks);
   let changeset: Changeset;
 
+  hooks.beforeEach(function () {
+    // TODO: fix how we perform measurements in situations where scale is applied on a container of animated stuff
```

Comment:
The default styling of the `#ember-testing` element has `scale(0.5)`, and our measurements don't account for parents with scale at the moment.